src/Cli.elm

```diff
@@ -774,16 +782,15 @@ printSuccessMessageAndWarnings ( outputPaths, warnings ) =
                 |> List.map (indentBy 4)
             , [ ""
               , ""
-              , "You'll also need " ++ String.Extra.toSentenceOxford requiredLinks ++ " installed. Try running:"
-              , ""
-              , indentBy 4 "elm install elm/http"
-              , indentBy 4 "elm install elm/json"
+              , "You'll also need " ++ toSentence requiredPackages ++ " installed. Try running:"
```

Comment:
Day dream idea, scan the local elm.json and see if these dependencies are already installed and only suggest installing if they aren't.

Reply:
... yeah, that would be nice

Reply:
Oh, I see you #145, nice